contracts/contracts/MerkleGrantRoundPayout.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.5;
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+
+/**
+ * @notice The MerkleGrantRoundPayout contract eligble grant owners to claim their
+ * match after round ends and funds have been loaded into the contract
+ * It allows grant owners to claim their funds in the given ERC20
+ * token upon verif merkleRoot
+ *
+ * This contract does the following
+ *  - funds transferred from GrantRound contract
+ *  - merkle root which contains how the funds are segrated and who can claim them
+ *  - emit Claimed event when grant owner has claimed funds
```

Comment:
I like the idea of explaining the whole flow in the comments here, but this is a bit unclear IMO. Perhaps something like:
```suggestion
 * This contract is intended to work as follows:
 *  - When a `GrantRound` is complete, compute the match payouts for that round
 *  - Generate a Merkle tree of the match payout results
 *  - Deploy an instance of this contract with the associated Merkle root
 *  - Transfer match funds from the `GrantRound` contract to this contract
*   - Users eligible for match payouts can use the `claim` or `batchClaim` method to claim their funds
```